stdlib/3/tkinter/tix.pyi

```diff
@@ -0,0 +1,305 @@
+    def add_separator(self, name: str, cnf: Dict[str, Any] = ..., **kw: Any) -> None: ...
+    def delete(self, name: str) -> None: ...
+    def disable(self, name: str) -> None: ...
+    def enable(self, name: str) -> None: ...
+
+class PopupMenu(TixWidget):
+    def __init__(self, master: Optional[tkinter.Widget], cnf: Dict[str, Any] = ..., **kw: Any) -> None: ...
+    def bind_widget(self, widget: tkinter.Widget) -> None: ...
+    def unbind_widget(self, widget: tkinter.Widget) -> None: ...
+    def post_widget(self, widget: tkinter.Widget, x: int, y: int) -> None: ...
+
+class Select(TixWidget):
+    def __init__(self, master: Optional[tkinter.Widget], cnf: Dict[str, Any] = ..., **kw: Any) -> None: ...
+    def add(self, name: str, cnf: Dict[str, Any] = ..., **kw: Any) -> tkinter.Widget: ...
+    def invoke(self, name: str) -> None: ...
+
+class StdButtonBox(TixWidget):
+    def __init__(self, master: Optional[tkinter.Widget] = ..., cnf: Dict[str, Any] = ..., **kw: Any) -> None: ...
+    def invoke(self, name: str) -> None: ...
+
+class DirList(TixWidget):
+    def __init__(self, master: Optional[tkinter.Widget], cnf: Dict[str, Any] = ..., **kw: Any) -> None: ...
+    def chdir(self, dir: str) -> None: ...
+
+class DirTree(TixWidget):
+    def __init__(self, master: Optional[tkinter.Widget], cnf: Dict[str, Any] = ..., **kw: Any) -> None: ...
+    def chdir(self, dir: str) -> None: ...
+
+class DirSelectDialog(TixWidget):
+    def __init__(self, master: Optional[tkinter.Widget], cnf: Dict[str, Any] = ..., **kw: Any) -> None: ...
+    def popup(self) -> None: ...
+    def popdown(self) -> None: ...
+
+class DirSelectBox(TixWidget):
+    def __init__(self, master: Optional[tkinter.Widget], cnf: Dict[str, Any] = ..., **kw: Any) -> None: ...
+
+class ExFileSelectBox(TixWidget):
+    def __init__(self, master: Optional[tkinter.Widget], cnf: Dict[str, Any] = ..., **kw: Any) -> None: ...
+    def filter(self) -> None: ...
+    def invoke(self) -> None: ...
+
+class FileSelectBox(TixWidget):
+    def __init__(self, master: Optional[tkinter.Widget], cnf: Dict[str, Any] = ..., **kw: Any) -> None: ...
+    def apply_filter(self) -> None: ...
+    def invoke(self) -> None: ...
+
+class FileEntry(TixWidget):
+    def __init__(self, master: Optional[tkinter.Widget], cnf: Dict[str, Any] = ..., **kw: Any) -> None: ...
+    def invoke(self) -> None: ...
+    def file_dialog(self) -> None: ...
+
+class HList(TixWidget, tkinter.XView, tkinter.YView):
+    def __init__(self, master: Optional[tkinter.Widget] = ..., cnf: Dict[str, Any] = ..., **kw: Any) -> None: ...
+    def add(self, entry: str, cnf: Dict[str, Any] = ..., **kw: Any) -> tkinter.Widget: ...
+    def add_child(self, parent: Optional[str] = ..., cnf: Dict[str, Any] = ..., **kw: Any) -> tkinter.Widget: ...
+    def anchor_set(self, entry: str) -> None: ...
+    def anchor_clear(self) -> None: ...
+    # FIXME: Overload, certain combos return, others don't
```

Comment:
Use `Any` as the return type for things like this to avoid having to write `assert return_value is not None` whenever the return value is used.